setools/policyrep/terule.pxi

```diff
@@ -152,6 +152,8 @@ cdef class AVRule(BaseTERule):
             raise RuleUseError("{0} is not {1} and is not in {1}".format(source, self.source))
         if target is not self.target and target not in self.target:
             raise RuleUseError("{0} is not {1} and is not in {1}".format(target, self.target))
+        if perms > self.perms:
```

Comment:
I don't think this is a complete check. Wouldn't this be false if `perms` and `self.perms` were disjoint? It seems like `if not perms <= self.perms:` is the right condition.

Reply:
You are right. I'll make the change.